libcni/api.go

```diff
@@ -28,25 +29,30 @@ type RuntimeConf struct {
 	Args        [][2]string
 }
 
+// NetworkConfig is network configuration network add/del operation
```

Comment:
for a network add/del operation
but this docstring doesn't add much value - can you elaborate?